tasks/infra/delete-rosa-task.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: delete-rosa
+spec:
+  description: 'Deletes ROSA.'
+  params:
+    - name: cluster-id
+      type: string
+    - name: aws-credentials
+      type: string
+    - name: region
+      type: string
+  steps:
+    - computeResources:
+        limits:
+          cpu: '100m'
+          memory: 64Mi
+      env:
+        - name: AWS_ACCOUNT_ID
+          valueFrom:
+            secretKeyRef:
+              key: AWS_ACCOUNT_ID
+              name: $(params.aws-credentials)
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              key: AWS_ACCESS_KEY_ID
+              name: $(params.aws-credentials)
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              key: AWS_SECRET_ACCESS_KEY
+              name: $(params.aws-credentials)
+      image: 'quay.io/trepel/kuadrant-pipelines:latest'
+      imagePullPolicy: Always
+      name: delete-rosa
+      script: |
+        #!/usr/bin/env bash
+        set -evo pipefail
+
+        # Use ocm.json created by rosa-login Task
+        export OCM_CONFIG=$(workspaces.shared-workspace.path)/ocm.json
+
+        # ROSA Cluster Deletion Trigger
+        rosa delete cluster --cluster=$(params.cluster-id) --region $(params.region) -y
```

Comment:
If you call the "watch command" it will wait until uninstall and than you can call additional cleanup commands:
```
rosa delete operator-roles -c <cluster_id> 
rosa delete oidc-provider -c <cluster_id>
(optional?) rosa delete account-roles --prefix ManagedOpenShift --mode auto 
```

Reply:
This can be done as part of future PR

Reply:
done, but it seems it is not really necessary